app/models/task.rb

```diff
@@ -817,6 +830,16 @@ def assignee_status_is_valid_on_create
     true
   end
 
+  def open_task_count_validation_required?
+    !Thread.current.thread_variable_get(:skip_duplicate_validation)
+  end
```

Comment:
```suggestion
  def check_for_open_task_type?
    !Thread.current.thread_variable_get(:skip_duplicate_validation)
  end
```

I'm also tempted to rename `skip_duplicate_validation` to `check_for_open_task_type` and reverse all the boolean logic, but that's a personal preference to ease readability for me.

Reply:
> I'm also tempted to rename skip_duplicate_validation to check_for_open_task_type and reverse all the boolean logic, but that's a personal preference to ease readability for me.
Hmm, interesting idea.  I will think this over and look at implementing it.